### ISSUE TITLE
Assistance Needed with DTU Cross-Generalization Test Reproduction

### ISSUE THREAD

**Issue by Youngju-Na** [donydchen/mvsplat#7]
Hi, sincerely appreciate sharing this amazing work!
I am currently working on reproducing the DTU cross-generalization test results as described in your recent publication. 

Despite my efforts to follow the experimental setup outlined in the paper, including ensuring the camera requirements are met (with normalized intrinsic parameters and cam2world matrices for extrinsic parameters), I've encountered difficulties in replicating the results presented in your paper, specifically the quality of the images.

For reference, here are the results I obtained:
![000004](https://github.com/donydchen/mvsplat/assets/45136186/b41ed912-3c30-49da-8f6c-3f2bd9048338)
![000005](https://github.com/donydchen/mvsplat/assets/45136186/f82fea5f-73e9-4048-9912-a5b6c244115d)

Would it be possible for you to share the specific dataloader used for the DTU evaluation or provide any guidance or recommendations that could aid in accurately evaluating the tests?

Thanks in advance.

Sincerely,

**Comment by donydchen** [comment]
Hi @Youngju-Na, thanks for your interest in our work.

We have not had time to clean the DTU data loader, but the DTU data loader is overall similar to the RE10K one. Since the DTU data conversion is quite different from RE10K, we just added a script (at commit f6c1714d78d709046bc728f999c6850759c5aaae) to help convert the DTU raw data to match the RE10K one. Some more instructions are detailed below,

* Download DTU following [this instruction](https://github.com/donydchen/matchnerf?tab=readme-ov-file#dtu-for-both-training-and-testing) (only need the 'dtu_training.rar' here).
* Run `python src/scripts/convert_dtu.py --input_dir PATH_TO_DTU --output_dir datasets/dtu` 

After that, just use a data loader similar to RE10K, and it should be good to go. Also, remember to set the (near, far) as (2.125, 4.525), and make sure that the context views have enough overlaps, e.g., you can try `{"context": [33, 31], "target": [32]}`. 

Feel free to let us know if you encounter any other difficulties. We will also find time to clean the DTU data loader and release it as soon as possible. Stay tuned.

**Comment by Youngju-Na** [comment]
Thanks! Problem solved.

The issue was with the setting near/far as you said and scaling down the extrinsic's translation part.
Thanks for your response. 
I'll close this issue.
